src/models/sentence_classifier/tokenizer.py

```diff
@@ -79,3 +79,8 @@ def tokenized_corpus_to_token_ids(
                 for text in tokenized_texts
             ]
         raise AttributeError("Vocabulary is not built yet!")
+
+    def __len__(self) -> int:
```

Comment:
maybe add more dunder methods to be able to access tokens in format (token, freq) or (token, idx(sorted))

Reply:
better do it a new branch with new pull request